iroh-gateway/src/utils.sh

```diff
@@ -0,0 +1,226 @@
+#!/bin/bash
```

Comment:
did you mean to add this?

Reply:
No, the shell scripts are a leftover of a `git add src/` ...

Reply:
fixed in latest version